docs/UserGuide.md

```diff
@@ -177,6 +181,102 @@ If yes: removes all students from the class list.
 clear
 ```
 
+### <a name="addses"></a>Adding a session : `addses`
+
+Adds a session to session list.
+
+```
+addses s/SESSION_NAME d/SESSION_DATE
+```
+
+:bulb: **Tip:**<div markdown="span" class="alert alert-primary">
+Two sessions cannot have the same name
+</div>
+
+Examples:
+* `addses s/tut1 d/12/7/2020`
+
+### <a name="deleteses"></a>Deleting a session : `deleteses`
+
+The user will be prompted to confirm their decision here.
+
+If yes: deletes the specified session from the session list.
+
+```
+deleteses INDEX
+```
+
+* Deletes the session at the specified `INDEX`.
+* The index refers to the index number shown in the displayed session list.
+* The index **must be a positive integer** 1, 2, 3, …
+* User will then be prompted for a confirmation input **`(yes/no)`**.
+
+Examples:
+* `deleteses s/tut1 d/12/7/2020` followed by `yes` deletes the 2nd session in the session list.
+
+### <a name="editses"></a>Editing a session : `editses`
+
+The user will be prompted to confirm their decision here.
+
+If yes: edits and updates the details of the specified session from the session list.
+
+```
+editses INDEX s/NEW_NAME d/NEW_DATE
+```
+
+* Edits the session at the specified `INDEX` with the specified details to be updated.
+* The index refers to the index number shown in the displayed session list.
+* The index **must be a positive integer** 1, 2, 3, …
+* User will then be prompted for a confirmation input **`(yes/no)`**.
+
+Examples:
+* `editsession 2 s/Tutorial1 d/10/10/2020` followed by `yes` edits the 2nd session in the session list with a new session name `Tutorial1` and a new session date `10/10/2020`.
+
+### <a name="enterses"></a>Entering a session : `enterses`
+
+Enters a session in the session list.
+
+```
+enter INDEX
+```
+
+* Enters the session at the specified `INDEX`.
+* The index refers to the index number shown in the displayed session list.
+* The index **must be a positive integer** 1, 2, 3, …
+
+Examples:
+* `enterses 1` enters the first session and allows you to use PARTICIPATE and PRESENCE commands.
+
+### <a name="participate"></a>Toggling participation status of student : `participate`
+
+Toggles the participation status of a student in the student list of the session.
+
+```
+PARTICIPATE INDEX_RANGE
```

Comment:
Shouldn't the command participate be in all small letters?

Reply:
yup thanks for spotting the mistake!